torchaudio/datasets/utils.py

```diff
@@ -129,7 +130,8 @@ def download_url(url: str,
 
     if resume and os.path.exists(filepath):
         mode = "ab"
-        local_size = os.path.getsize(filepath)
+        local_size = os.path.getsize(filepath)  # type: Optional[int]
```

Comment:
First, we need to check that the rest of pytorch is following that style. In the meanwhile, I suggest to skip this test in mypy.

By the way, since torchaudio's master branch only support python 3.6 and up, we could use
`local_size: Optional[int] = os.path.getsize(filepath)` instead.